api/test/wfl/integration/jdbc_test.clj

```diff
@@ -7,17 +7,24 @@
 (use-fixtures :once fixtures/temporary-postgresql-database)
```

Comment:
Is this to support the new `:watchers` column?

Reply:
hmm, no, this was here for a while 🤔

Reply:
But why then? ;-)